bin/compile

```diff
@@ -191,7 +191,7 @@ def build_spirv_toolkit_and_level_zero(rebuild=False):
 
     if (rebuild or build):
         os.chdir(spirv_tool_kit)
-        subprocess.run(["git", "pull", "origin", "master"])
```

Comment:
I think this needs to be reverted

Reply:
We keep this until we merge the code from the Beehive SPIR-V Toolkit